### PR TITLE
NNCP: get NNCE for node

### PR DESCRIPTION
**Pull request by myakove**
##### Short description:
Get NNCE from NNCP by node.

### CODE REVIEW

File: ocp_resources/node_network_configuration_policy.py [hunk 1]
```diff
@@ -440,6 +440,11 @@ def nnces(self):
                 nnces.append(nnce)
         return nnces
 
+    def node_nnce(self, node_name):
+        for nnce in self.nnces:
+            if nnce.instance.metadata.labels["nmstate.io/node"] == node_name:
```
Comment (geetikakay) [inline ocp_resources/node_network_configuration_policy.py:445]: maybe use : nnce.labels['nmstate.io/node'] directly

File: ocp_resources/node_network_configuration_policy.py [hunk 2]
```diff
@@ -440,6 +440,11 @@ def nnces(self):
                 nnces.append(nnce)
         return nnces
 
+    def node_nnce(self, node_name):
+        for nnce in self.nnces:
```
Comment (rnetser) [inline ocp_resources/node_network_configuration_policy.py:444]: return [nnce for nnce in self.nnces if  nnce.labels["nmstate.io/node"] == node_name]

**Review by rnetser — CHANGES_REQUESTED** [review]

**Review by rnetser — APPROVED** [review]

File: ocp_resources/node_network_configuration_policy.py [hunk 3]
```diff
@@ -440,6 +440,11 @@ def nnces(self):
                 nnces.append(nnce)
         return nnces
 
+    def node_nnce(self, node_name):
+        return [
+            nnce for nnce in self.nnces if nnce.labels["nmstate.io/node"] == node_name
```
Comment (geetikakay) [inline ocp_resources/node_network_configuration_policy.py:445]: @myakove @rnetser I am bit confused here . why we make return output as list. if policy name is known, i should just be getting one enactment (NodeNetworkConfigurationEnactment object. )

Comment (myakove) [reply ocp_resources/node_network_configuration_policy.py:445]: You are right, I will fix it.

**Pull request merged**